spec/features/deck_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+require 'deck'
+
+describe "My deck of cards" do
+  it "should have 52 cards" do
+    my_deck = Deck.new
+    my_deck.cards.size.must_equal(52)
+  end
+
+  it "should shuffle the deck" do
+    unshuffled = Deck.new
+    shuffled = Deck.new
+    shuffled.shuffle!
+    unshuffled.cards.first.to_s.wont_equal(shuffled.cards.first.to_s)
```

Comment:
Occasionally, this assertion will fail, even though the deck is shuffled (if the first card ends up shuffled to the same place). Have any other ideas for how to test this?